optuna/trial.py

```diff
@@ -148,6 +158,84 @@ def _init_relative_params(self):
             self.study, trial, self.relative_search_space
         )
 
+    def suggest_float(self, name, low, high, *, log=False):
+        # type: (str, float, float, bool) -> float
+        """Suggest a value for the floating point parameter.
+
+        Note that this is a wrapper method for :func:`~optuna.trial.Trial.suggest_uniform`
+        and :func:`~optuna.trial.Trial.suggest_loguniform`.
+
+        .. versionadded:: 1.3.0
+
+        .. seealso::
+            Please see also :func:`~optuna.trial.Trial.suggest_uniform` and
+            :func:`~optuna.trial.Trial.suggest_loguniform`.
+
+        Example:
+
+            Suggest a momentum and learning rate for neural network training.
+
+            .. testsetup::
+
+                import numpy as np
+                import optuna
+                from sklearn.model_selection import train_test_split
+                from sklearn.neural_network import MLPClassifier
+
+                np.random.seed(seed=0)
+                X = np.random.randn(200).reshape(-1, 1)
+                y = np.random.randint(0, 2, 200)
+                X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+
+
+            ..testcode::
+
+                def objective(trial):
+                    momentum = trial.suggest_float('momentum', 0.0, 1.0)
+                    learning_rate_init = trial.suggest_float('learning_rate_init',
+                                                             1e-5, 1e-3, log=True)
+                    clf = MLPClassifier(hidden_layer_sizes=(100, 50), momentum=momentum,
+                                        learning_rate_init=learning_rate_init,
+                                        solver='sgd', random_state=0)
+                    clf.fit(X_train, y_train)
+
+                    return clf.score(X_test, y_test)
+
+                study = optuna.create_study(direction='maximize')
+                study.optimize(objective, n_trials=3)
+
+        Args:
+            name:
+                A parameter name.
+            low:
+                Lower endpoint of the range of suggested values. ``low`` is included in the range.
+            high:
+                Upper endpoint of the range of suggested values. ``high`` is excluded from the
+                range.
+            log:
+                A flag to sample the value from the log domain or not.
+                If ``log`` is true, the value is sampled from the range in the log domain.
+                Otherwise, the value is sampled from the range in the linear domain.
+                See also :func:`suggest_uniform` and :func:`suggest_loguniform`.
+
+        Returns:
+            A suggested float value.
+        """
+
+        if log:
```

Comment:
The comment https://github.com/optuna/optuna/pull/1021/files#r396901171 was marked as resolved but it seems the suggestion hasn't been applied.

Reply:
It seems I have resolved this unintentionally, sorry.
Applied in 4669c1b.

Reply:
Thanks!